jakarta-eclipse/src/org/jakarta/lsp4e/JakartaJavaHoverProvider.java

```diff
@@ -0,0 +1,35 @@
+package org.jakarta.lsp4e;
+
+import org.eclipse.jdt.ui.text.java.hover.IJavaEditorTextHover;
+import org.eclipse.jface.text.IRegion;
+import org.eclipse.jface.text.ITextViewer;
+import org.eclipse.lsp4e.operations.hover.LSPTextHover;
+import org.eclipse.ui.IEditorPart;
+
+@SuppressWarnings("restriction")
+public class JakartaJavaHoverProvider implements IJavaEditorTextHover {
+	private LSPTextHover lspTextHover;
+	
+	public JakartaJavaHoverProvider() {
+		super();
+		lspTextHover = new LSPTextHover();
+	}
+
+	@Override
+	public String getHoverInfo(ITextViewer arg0, IRegion arg1) {
```

Comment:
public String getHoverInfo(ITextViewer textViewer, IRegion region) {

Also remove the TODO messages